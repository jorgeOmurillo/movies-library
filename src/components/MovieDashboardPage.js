import React from 'react';
import MoviesListFilters from '../components/MoviesListFilters';
import MoviesSummary from '../components/MoviesSummary';
import MovieList from '../components/MovieList';

const MovieDashboardPage = () => (
  <div>
    <MoviesSummary />
    <MoviesListFilters />
    <MovieList />
  </div>
);

export default MovieDashboardPage;
