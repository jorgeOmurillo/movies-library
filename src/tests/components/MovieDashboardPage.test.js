import React from 'react';
import { shallow } from 'enzyme';
import MovieDashboardPage from '../../components/MovieDashboardPage';

test('should render DashboardPage correctly', () => {
  const wrapper = shallow(<MovieDashboardPage />);

  expect(wrapper).toMatchSnapshot();  
});
