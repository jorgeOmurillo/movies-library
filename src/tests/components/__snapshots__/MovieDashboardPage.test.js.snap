// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboardPage correctly 1`] = `
<div>
  <Connect(MoviesSummary) />
  <Connect(MoviesListFilters) />
  <Connect(MovieList) />
</div>
`;
